src/js/pages/Values/OneValue.jsx

```diff
@@ -264,20 +267,55 @@ class OneValue extends Component {
             <FilterChoices>
               <Chip
                 key="forThisElectionKey"
-                id = "forThisElection"
+                id="forThisElection"
                 label={<span style={showEndorsersForThisElection ? { fontWeight: 600 } : {}}>For This Election</span>}
                 className={showEndorsersForThisElection ? classes.selectedChip : classes.notSelectedChip}
                 component="div"
-                onClick={() => this.changeListModeShown('voterGuidesForThisElection')}
+                // Yousra Elzamzami - WV-1135
+                onClick={() => {
+                  const {pageName, pageType} = lookupPageNameAndPageTypeDict(window.location.pathname);
```

Comment:
Please deconstruct like this (instead of using `window.location.pathname`):
const { location: { pathname: currentPathname } } = window;

Also, make sure to include spaces like this:
const { pageName, pageType } =
If you execute this in your terminal window within the WebApp folder, you'll get the ESLint warnings and errors:
`npm run lint`